### DISCUSSION TITLE
fail2ban setup on remote machine with syslog

### DISCUSSION THREAD

**Discussion by Stephan-4711** [hassio-addons/addon-bitwarden#286] [Q&A]
Hi,

I'm trying to setup fail2ban on my nginx system, which is an extra LXC on my proxmox server.
So I need to enable a syslog server for vaultwarden.
The manuals I found say, that I need to edit the docker-compse.yml but a find didn't help me finding the file.
can you tell me, how to add a syslogserver?

**Comment by frenck** [comment]
I think you are in the wrong place.

This is the Home Assistant add-on, which cannot be used with Docker directly.

../Frenck

**Reply by Stephan-4711** [reply]
No I'm trying to activate fail2ban in your addon. But I use an extra nginx lxc on my proxmox not the provided solution by homeassistant

**Reply by frenck** [reply]
Sorry, it sounds like you are better of running Vaultwarden in a different way in that case.

**Reply by Stephan-4711** [reply]
Then I can't participate of the google drive backup home assistant addon to backup my vaultwarden.
